spec/spec_helper.rb

```diff
@@ -14,6 +14,7 @@
 #
 # See https://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  require 'capybara/rspec'
```

Comment:
- [ ] I am unable to run your tests successfully, because you are missing the `capybara` gem in your Gemfile. Kindly add the `capybara` gem to the development and test group.

  ```
  group :development, :test do
    ...
    gem 'capybara'
  end
  ```

  ![image](https://github.com/AlyAtalla/blog-app/assets/36542986/edf6ce3d-216f-42d9-803f-0dfaa71711fb)